packages/ia-userlist-settings/src/ia-user-list-settings.ts

```diff
@@ -38,10 +44,13 @@ export class IAUserListSettings extends LitElement {
 
   @query('#private') private listPrivate: HTMLInputElement;
 
+  @state() private disableSaveButton = false;
```

Comment:
This is specific change to disable save button after initial click and enable if any error occure.